cms/templates/partials/text-video-section.html

```diff
@@ -12,8 +13,12 @@ <h1 class="text-uppercase">{{ page.title }}</h1>
       <div class="col-lg-6">
         {% if page.video_url %}
           <div class="video-holder">
-            <video id="tv-video" controls data-source="{{ page.video_url }}">
-            </video>
+              {% embed page.video_url as youtube_video %}
```

Comment:
just a question, so the `embed` tag will take care of `youtube urls` ?

Reply:
yes